src/routes/(modeles-services)/services/[slug]/orienter/demande/orientation-form.svelte

```diff
@@ -0,0 +1,274 @@
+      <p class="mb-s0 text-f14 text-gray-text">
+        Personne en charge de l‘accompagnement et du suivi professionnel de la
+        situation du bénéficiaire. Si ce n‘est pas vous, veuillez modifier les
+        informations ci-dessous.
+      </p>
+    </Notice>
+
+    <div class="flex flex-row justify-items-stretch gap-s24">
+      <div class="flex-1">
+        <BasicInputField
+          id="referentLastName"
+          placeholder=""
+          bind:value={$orientation.referentLastName}
+          vertical
+        />
+      </div>
+      <div class="flex-1">
+        <BasicInputField
+          id="referentFirstName"
+          placeholder=""
+          bind:value={$orientation.referentFirstName}
+          vertical
+        />
+      </div>
+    </div>
+    <div class="flex flex-row justify-items-stretch gap-s24">
+      <div class="flex-1">
+        <BasicInputField
+          id="referentPhone"
+          type="tel"
+          description="Format attendu : 0123456789"
+          bind:value={$orientation.referentPhone}
+          vertical
+        />
+      </div>
+      <div class="flex-1">
+        <BasicInputField
+          id="referentEmail"
+          type="email"
+          description="Format attendu : nom@domaine.fr"
+          bind:value={$orientation.referentEmail}
+          vertical
+        />
+      </div>
+    </div>
+  </Fieldset>
+
+  <Fieldset title="Le ou la bénéficiaire">
+    <div class="flex flex-row justify-items-stretch gap-s24">
+      <div class="flex-1">
+        <BasicInputField
+          id="beneficiaryLastName"
+          placeholder=""
+          bind:value={$orientation.beneficiaryLastName}
+          vertical
+        />
+      </div>
+      <div class="flex-1">
+        <BasicInputField
+          id="beneficiaryFirstName"
+          placeholder=""
+          bind:value={$orientation.beneficiaryFirstName}
+          vertical
+        />
+      </div>
+    </div>
+
+    <BasicInputField
+      id="beneficiaryAvailability"
+      type="date"
+      description=""
+      bind:value={$orientation.beneficiaryAvailability}
+      vertical
+    >
+      <p slot="description" class="legend italic">
+        Date à partir de laquelle la personne est disponible. <br />
+        Format attendu : JJ/MM/AAAA (par exemple, 17/01/2023 pour 17 janvier 2023)
+      </p>
+    </BasicInputField>
+
+    {#if $orientation.requirements.length || $orientation.situation.length}
+      <div class="rounded-md bg-info-light px-s20 py-s20">
+        <Accordion
+          title="Profil et critères du ou de la bénéficiaire"
+          subTitle="Récapitulatif des critères que vous avez confirmé à l’étape précédente."
+          titleLevel="h3"
+          noTitleMargin
+          titleClass="text-f18"
+          expanded={true}
+        >
+          <div class="mt-s20">
+            {#if $orientation.situation.length}
+              <h4 class="mb-s6 text-gray-text">
+                Profil de votre bénéficiaire :
```

Comment:
```suggestion
                Profil de votre bénéficiaire&nbsp;:
```
essaye de faire une passe sur les espaces insécables manquant

Reply:
il en manque encore plein